tools/configure_server_settings.rb

```diff
@@ -2,21 +2,50 @@
 require File.expand_path("../config/environment", __dir__)
 require 'trollop'
 
+TYPES = %w(string integer boolean symbol float).freeze
+
 opts = Trollop.options(ARGV) do
   banner "USAGE:   #{__FILE__} -s <server id> -p <settings path separated by a /> -v <new value>\n" \
-         "Example: #{__FILE__} -d -s 1 -p reporting/history/keep_reports -v 42"
+         "Example (String): #{__FILE__} -s 1 -p reporting/history/keep_reports -v 42\n" \
```

Comment:
Using a duration is probably a more helpful example for a string:
`Example (String): #{__FILE__} -s 1 -p reporting/history/keep_reports -v '3.months'`

Reply:
👍 yeah - when I was testing moving from 6.months -> 42 caught me off guard.